src/components/molecules/Settings/Account/PasswordModal/index.tsx

```diff
@@ -27,17 +27,24 @@ type Props = {
 
 const PasswordModal: React.FC<Props> = ({ isVisible, onClose, hasPassword, updatePassword }) => {
   const intl = useIntl();
+  const theme = useTheme();
 
-  const [currentPassword, setCurrentPassword] = useState("");
   const [password, setPassword] = useState("");
   const [passwordConfirmation, setPasswordConfirmation] = useState("");
   const [disabled, setDisabled] = useState(true);
 
+  const handleClose = useCallback(() => {
+    setPassword("");
+    setPasswordConfirmation("");
+    onClose?.();
+  }, [onClose]);
+
   const save = useCallback(() => {
     if (password === passwordConfirmation) {
```

Comment:
If `password` doesn't match `passwordConfirmation` front-end should show the validation error inside of the form. 
Does this modal have any validation?

Reply:
None yet. I was treating this as a hotfix since you said users already have passwords they want to change. It also doesn't have any regex to make sure it is of length, etc etc which I think eventually it should have. 
The reason I didn't do in this PR is because I am not comfortable with validation/regex and knew it would take me time to look into options/how to do it. 
Let me know if you want that included in this PR.

Reply:
@KaWaite OK thank you. 👍 
You don't need to do that as a part of this PR.
Plz, create the issue `Add client-side validation on password modal` and write the description.
Maybe it'll be good for the first issue for contributers.